codegen/stub_api.py

```diff
@@ -14,6 +14,13 @@
 from typing import Any, List, Optional
 from xml.etree import ElementTree
 
+UTILS_MAP = {
```

Comment:
This is required for fixing the utils. I am using a dictionary just in case module name and class name are not the same